README.md

```diff
@@ -0,0 +1,41 @@
+# onebusaway-travel-behavior-analysis
+Python utilities to process and analyze travel behavior data collected by OneBusAway and exported as csv file by oba-firebase-export.
+
+## matchAndMerge utility
+This utility matches ground truth data activities in excel format with activities data generated by oba-firebase-export in csv format. The algorithm will match each activity in the ground truth dataset with the nearest activity in the oba-firebase-export generated file. 
+
+## Build 
+tbd
+
+## Run
+To run the matchAndMerge script use `python` command and pass the desired arguments:
+`python matchAndMerge.py`
+
```

Comment:
Could you please add a `### InputData` section here, with a description of the input data?

For the OBA data you can just point to the export software GitHub repo (I think we have a description there?).

For the ground truth data, it would be good if you include a sample file with dummy data so it's clear what the tool expected. In this section, please also document about handling the rounding, timezone info, and anything else you think is important.